docs/ledger.md

```diff
@@ -0,0 +1,77 @@
+<!---
+parent:
+  order: false
+--->
+# Kava Ledger App
+
+[Ledger Nano S](https://www.ledger.com/products/ledger-nano-s) is a hardware wallet that enables users to securely access and use cryptocurrencies. Hardware wallets are considered very secure for the storage of a user’s private keys in the blockchain world. Your digital assets are safe even when using an infected (or untrusted) PC.
+
+## Before you begin
+1. [Initialize](https://support.ledgerwallet.com/hc/en-us/articles/360000613793) your Ledger Nano S.
+2. Update your Ledger device to the latest [firmware](https://support.ledgerwallet.com/hc/en-us/articles/360002731113-Update-Ledger-Nano-S-firmware).
+3. [Install](https://support.ledger.com/hc/en-us/articles/360006395553) Ledger Live.
+4. Install Google Chrome.
+
+## Install the Kava app on your Ledger Device
+1. Open the Manager in Ledger Live.
+2. Connect and unlock your Ledger Nano S.
+3. If asked, allow the Manager on your device by pressing the right button.
+4. Find Kava in the app catalog.
+5. Click the Install button of the Kava app.
+6. Search for Binance in the app catalog.
+7. Click the Install button of the Binance app.
+
+## Using Ledger device with Kava Web App
+
+Kava's web application available at [app.kava.io](https://app.kava.io/) supports ledger interaction with the Kava blockchain.
+1. Open and unlock the Kava ledger app using your pin.
+2. From the Connect Wallet display, click Ledger.
+3. Confirm your ledger address by clicking Show on device.
+4. View balances by clicking Balances.
+
+### Loading funds onto Kava
+1. From the Balances display, click Load.
+2. Close the Kava ledger app and open the Binance ledger app.
+3. Click Connect Binance App.
+4. On the Fund Account screen enter the amount and recipient.
+5. Click Fund.
+6. Review and sign the transaction on your ledger device.
```

Comment:
From the requirements

> - How to receive crypto assets
>    - Document all the steps.
>    - Explicitly instruct to verify receive addresses on the device.
> - How to send crypto assets
>    - Document all the steps and form fields to fill.
>    - Explicitly instruct to verify transaction details on the device display, before confirming and signing it.


I think a separate section needs to be added in order to send + receive coins to an address (eg send KAVA or USDX). Can be via Lunie or other

Reply:
We can’t use Lunie’s web wallet to send crypto using the Kava ledger app, we'll need to add this functionality directly to the web app as even after we update the middleware Lunie isn't going to replace their own library with our fork.